.travis.yml

```diff
@@ -12,6 +12,10 @@ before_script:
   - ./cc-test-reporter before-build
 script:
   - npm test
+  - npm run benchmark
```

Comment:
This is great. thank you! Any README updates required?

Reply:
yes it does

Reply:
updates PULL_REQUEST TEMPLATE, didn't found any othee reference to npm test...